src/locales/fr.yml

```
@@ -112,3 +115,125 @@ homepage:
+    tab-1:
+      title: Gestion d'équipe
+      description: 'Centralisez toutes les informations importantes concernant votre équipe au même endroit, et gérez le quotidien de ses membres (ex : gestion de congés ou calendrier de présence).'
+    tab-2:
+      title: Gestion de tâches
+      description: 'Organisez, assignez et suivez l’avancement de chaque action dans un seul tableau.'
+    tab-3:
+      title: Collecte d’informations
+      description: 'Grist peut vous aider à faire remonter des données terrain depuis partout en France, dans un document unique avec une gestion d’accès et de droits avancée. Vous pouvez également créer des formulaires publics.'
+    tab-4:
+      title: Évènementiel
+      description: 'Collectez des inscriptions à vos événements, et gérez les invités ou la logistique facilement.'
+    tab-5:
+      title: Gestion budgétaire
+      description: 'Créez vos propres tableaux de bord, pour calculer et suivre votre budget.'
+  cta-1:
+    button-1:
+      href: 'https://grist.numerique.gouv.fr/'
+      label: Commencer à utiliser Grist
+    button-2:
+      href: '#contact'
+      label: Nous contacter
+  mosaic:
+    block-1:
+      title: Collaborez avec votre équipe, en toute fluidité
+      description: 'Avec ses fonctionnalités avancées de collecte et de gestion de données, Grist simplifie la collaboration au sein de votre équipe, ou avec d’autres administrations.'
+      tag: Collaborer
+    block-2:
+      title: Prenez des décisions basées sur vos données
+      description: 'Grâce à ses tableaux de bord modulables et interactifs, et ses cas d’usage variés, Grist vous offre une vision claire de vos données, pour vous aider à orienter vos choix.'
+      tag: Orienter
+    block-3:
+      title: Gagnez en productivité
+      description: 'En centralisant toutes vos données, en éliminant la double saisie et en utilisant notre assistant IA, Grist vous fera gagner un temps précieux.'
+      tag: Productivité
+    block-4:
+      title: Garantissez votre souveraineté et votre sécurité
+      description: 'Hébergé en France, open-source et certifié SecNumCloud, Grist protège vos données et vous assure souveraineté et conformité.'
+      tag: Souveraineté
+  cards:
+    card-1:
+      title: Intégrez vos propres outils, sans difficultés
+      description: 'Avec ses widgets sur mesure, ses possibilités d’intégration et ses fonctionnalités de personnalisation, Grist s’adapte à votre écosystème et à vos besoins spécifiques.'
+    card-2:
+      title: Assurez l’intégrité de vos données
+      description: 'Avec ses contrôles intégrés et le suivi des versions, Grist prévient les erreurs et garantit des données fiables.'
+    card-3:
+      title: Embarquez rapidement
+      description: 'Fini les longs processus IT ! Accédez gratuitement à notre logiciel homologué par la DINUM, et profitez de modèles prêts à l’emploi et d’une communauté active pour démarrer et monter en compétences rapidement.'
```

```suggestion
      description: 'Accédez gratuitement à notre logiciel homologué par la DINUM, et profitez de modèles prêts à l’emploi et d’une communauté active pour démarrer et monter en compétences rapidement.'
```

C'est un peu chaud de taper sur les DSI comme ça